chainstate/src/detail/median_time.rs

```diff
@@ -46,7 +46,10 @@ mod test {
         detail::tx_verification_strategy::DefaultTransactionVerificationStrategy, BlockSource,
         Chainstate, ChainstateConfig,
     };
-    use common::time_getter::TimeGetter;
+    use common::{
```

Comment:
I think you can change `primitives::time::{self}` to `primitives::time`

Reply:
Right, thanks.